detectable/src/main/java/com/blackduck/integration/detectable/detectables/gradle/inspection/parse/GradleReportTransformer.java

```diff
@@ -122,10 +123,29 @@ private boolean codeLocationAlreadyCreatedFor(String subProjectName) {
     }
 
     private CodeLocation createCodeLocationForSubProject(GradleReport rootReport, DependencyGraph subProjectGraph, String subProjectName) {
-        ExternalId projectId = ExternalId.FACTORY.createMavenExternalId(rootReport.getProjectGroup(), subProjectName, rootReport.getProjectVersionName());
+        ExternalId projectId;
```

Comment:
nit: seems like you can save a few lines by declaring `projectId` and `subProjectPath` where they are instantiated.
This is more of a personal preference.

Reply:
Done, thank you